ReleaseNotes.md

```diff
@@ -53,11 +76,62 @@ We now don't make this check, as creating a topic is idempotent, this results in
 
 ## Upgrade instructions
 
-TODO
-Show logging
-Show packages changes with csproj snippet
-Show IAM changes
-Show backoff strategy stuff
+Let's take the following snippet that uses JustSaying v4, and let's update it to v5:
+```csharp
+var bus = CreateMeABus
+            .InRegion(RegionEndpoint.EUWest1.SystemName)
+            .WithMonitoring(_monitoring)
+            .ConfigurePublisherWith(c =>
+                {
+                    c.PublishFailureBackoffMilliseconds = 1;
+                    c.PublishFailureReAttempts = 3;
+                })
+            .WithSnsMessagePublisher<GenericMessage>()
+            .WithSqsTopicSubscriber()
+            .IntoQueue("queuename")
+            .ConfigureSubscriptionWith(cfg =>
+                {
+                    cfg.MessageRetentionSeconds = 60;
+                    cfg.InstancePosition = 1;
+                    cfg.OnError = _globalErrorHandler;
+                })
+            .WithMessageHandler(_handler);
+```
+
+After you update the JustSaying pacakge, be sure to tidy up any `app.config` assembly binding redirects that mention `JustSaying.AwsTools` and `JustSaying.Messaging` as these no longer exist, NuGet should do this for you.
+
+The snippet above won't compile as we need to configure a logger before do any further configuration:
+```csharp
+// You might use a LoggerFactory throughout your application,
+//   or you might be using your logging library abstraction, and this is just a bridge
+var loggerFactory = new LoggerFactory();
+loggerFactory.AddMyFavouriteLoggingLibrary(); // Example: .AddSerilog()
+
+var bus = CreateMeABus
+            .WithLogging(loggerFactory) // <- NEW
+            .InRegion(RegionEndpoint.EUWest1.SystemName)
+            ...
+```
+
+### IAM permissions
+In this release the AWS APIs that are called have changed slightly with regards to SNS, so it might be worth checking that you have the required IAM permissions set up.
+
+Here are the actions requred if you are both publishing and subscribing.
+```
+sns:CreateTopic,
```

Comment:
The fusspot in me suggests sorting these A-Z so it's easier to compare to another list if someone's trying to check theirs vs. this.